README.md

```diff
@@ -100,55 +86,34 @@ Optional environment variables:
 
 And updated list of models is avaiable (here)[https://docs.perplexity.ai/guides/model-cards]
 
-#### Claude Desktop
-
-Add this tool as a mcp server by editing the Claude config file.
+### Cursor & Claude Desktop Installation
 
-On MacOS: `~/Library/Application\ Support/Claude/claude_desktop_config.json`
-On Windows: `%APPDATA%/Claude/claude_desktop_config.json`
+Add this tool as a mcp server by editing the Cursor/Claude config file.
 
 ```json
   "perplexity-mcp": {
     "env": {
       "PERPLEXITY_API_KEY": "XXXXXXXXXXXXXXXXXXXX",
       "PERPLEXITY_MODEL": "sonar"
     },
-    "command": "uv",
+    "command": "uvx",
     "args": [
-      "run",
       "perplexity-mcp"
     ]
   }
 ```
 
-To verify the server is working. Open the Claude client and use a prompt like "search the web for news about openai in the past week". You should see an alert box open to confirm tool usage. Click "Allow for this chat".
-
-  <img width="600" alt="mcp_screenshot" src="https://github.com/user-attachments/assets/922d8f6a-8c9a-4978-8be6-788e70b4d049" />
-
-### Cursor Installation
-
-You can also use perplexity-mcp as MCP server in Cursor.
-
-Since there no way to set an the the PERPLEXITY_API_KEY in Cursor, you'll have to create a simple wrapper script to set the environment variable.
-
-Example wrapper script:
-
-```
-#!/bin/bash
-
-# Set the your Perplexity API key
-export PERPLEXITY_API_KEY="pplx-XXXXXXXXXXXX"
+#### Cursor
+- On MacOS: `/Users/your-username/.cursor/mcp.json`
+- On Windows: `C:\Users\your-username\.cursor\mcp.json`
```

Comment:
Note: I'm not sure if the windows path is correct. I don't have access to a windows machine to confirm